src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSFilter.java

```diff
@@ -14,12 +15,11 @@
 import org.broadinstitute.hellbender.tools.spark.sv.utils.SVUtils;
 import org.broadinstitute.hellbender.tools.spark.utils.ReadFilterSparkifier;
 import org.broadinstitute.hellbender.tools.spark.utils.ReadTransformerSparkifier;
-import org.broadinstitute.hellbender.transformers.BaseQualityClipReadTransformer;
-import org.broadinstitute.hellbender.transformers.BaseQualityReadTransformer;
-import org.broadinstitute.hellbender.transformers.DUSTReadTransformer;
+import org.broadinstitute.hellbender.transformers.*;
```

Comment:
I don't feel strongly about this, but maybe better to import the classes individually so that dependencies can more easily be managed.